src/desktop/components/search_bar/view.coffee

```diff
@@ -198,12 +198,15 @@ module.exports = class SearchBarView extends Backbone.View
 
   selectResult: (e, model) ->
     return @emptyItemClick() unless model
-    analyticsHooks.trigger 'search:item:click',
-      query: @query
-      item_number: model.collection.models.findIndex( (result) -> return result.id == model.id ) + 1
-      item_type: model.get('display_model')
-      item_id: model.get('_id')
-      destination_path: "#{sd.APP_URL}#{model.href()}"
+
+    if model.collection and model.collection.models
+      analyticsHooks.trigger 'search:item:click',
```

Comment:
I wonder if we want to still send an event in this case, just with an empty `item_number`? 

In any case this seems pretty minor so feel free to merge at will 👍 .